modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -286,15 +301,67 @@ private List<ObjectViewSortColumn> _addObjectViewSortColumns(
 			});
 	}
 
-	private void _validate(
+	private void _validate(long objectViewId, long objectDefinitionId)
+		throws PortalException {
+
+		ObjectView objectView = objectViewPersistence.fetchByODI_DOV_First(
+			objectDefinitionId, true, null);
+
+		if ((objectView != null) &&
+			(objectView.getObjectViewId() != objectViewId)) {
+
+			throw new DefaultObjectViewException(
+				"There can only be one default object view");
+		}
+	}
+
+	private void _validateObjectViewColumn(
```

Comment:
Since the parameter is a list (List<ObjectViewColumn>), could you rename the method to the plural (_validateObjectViewColumns) ?

Reply:
Ok